index.bs

```diff
@@ -700,22 +721,56 @@ dictionary SensorOptions {
 
 A {{Sensor}} object has an associated [=sensor=].
 
+
+### Sensor task source ### {#task-source}
+
 Each {{Sensor}} object has a [=task source=]
 called a <dfn>sensor task source</dfn>, initially empty.
-A [=sensor task source=] can be enabled or disabled,
-and is initially enabled.
-When enabled,
+
+Issue(212): 
+
+A [=sensor task source=] can be enabled or disabled.
+
+To determine its initial state,
+run the steps to [=update the state of the sensor task source=].
+
+When the [=sensor task source=] is enabled,
 the [=event loop=] must use it as one of its [=task sources=].
 The [=task source=] for the [=tasks=] mentioned in this specification
 is the [=sensor task source=].
 
-When the [=visibility state=] of the [=Document=] in
+When the [=visibility state=] of the [=active document=] in
 the [=top-level browsing context=] changes,
-let |current_visibility_state| be the result of running
-the [=steps to determine the visibility state=] of the [=Document=].
-If |current_visibility_state| is "visible",
-enable the sensor task source,
-otherwise, disable it.
+the user agent must [=update the state of the sensor task source=].
+
+When an element that is the [=DOM anchor=] of a [=focusable area=] in any [=browsing context=] [=gains focus=]
+the user agent must [=update the state of the sensor task source=].
```

Comment:
@tobie So, can I proceed with #206 and then #213 can be rebased? I'm not sure if this PR can be fixed easily. Especially focusing / unfocusing is quite complex when origin is used as a criteria.